salt/modules/debbuild.py

```diff
@@ -452,20 +455,13 @@ def build(runas,
     return ret
 
 
-def make_repo(repodir,
-              keyid=None,
-              env=None,
-              use_passphrase=False,
-              gnupghome='/etc/salt/gpgkeys',
-              runas='root'):
+def make_repo(repodir, keyid=None, env=None, use_passphrase=False, gnupghome='/etc/salt/gpgkeys', runas='root'):
```

Comment:
Will do so in future, pylint didn't pill it up so missed it in rush to get in before vacation and tagging deadline.